1_fetch.yml

```diff
@@ -29,51 +29,58 @@ targets:
 
   1_fetch:
     depends:
-      - 1_fetch/out/streamdata.rds.ind
-      - 1_fetch/out/storm_track.zip.ind
       - view_polygon
+      - 1_fetch/out/storm_track.zip.ind
       - 1_fetch/out/all_sites.rds.ind
+      - 1_fetch/out/streamdata.rds.ind
+      - 1_fetch/out/precip_spatial.rds.ind
+      - 1_fetch/out/precip_values.rds.ind
 
   # -- config --
   dates:
     command: viz_config[[I('dates')]]
   stream_params:
     command: yaml.load_file('1_fetch/cfg/stream_params.yml')
 
-  # -- download stormtrack --
-  storm_track_cfg:
-    command: viz_config[I('storm_code')]
-  1_fetch/out/storm_track.zip.ind:
-    command: fetch_storm_track(ind_file=target_name, cfg=storm_track_cfg)
-  1_fetch/out/storm_track.zip:
-    command: gd_get('1_fetch/out/storm_track.zip.ind')
-
   # -- get spatial extent and data --
 
   view_config:
     command: viz_config[I(c('bbox', 'projection', 'width', 'height'))]
 
+  1_fetch/out/view_polygon.rds.ind:
+    command: post_view_polygon(target_name, view_config)
+  1_fetch/out/view_polygon.rds:
+    command: gd_get('1_fetch/out/view_polygon.rds.ind')
+  view_polygon:
+    command: get_view_polygon('1_fetch/out/view_polygon.rds.ind')
```

Comment:
I know that you needed to make secondary_geoms and focus_geoms have files and indicator files in order to get around the OS issues with rebuilding, but does that mean we can't do the third step here for those and get them back as an object to pass around?

Reply:
i think we could. but is it worth it for secondary_geoms and focus_geoms? those get used a lot less than view_polygon. that's why i didn't do it for all three.

Reply:
taking it to the extreme, every file in this whole project could have that third step. but then everybody would always need to download every file, reducing some of the benefit of the shared cache. so i think drawing the line closer to very few cases like `view_polygon` is a good idea.